script/circleci

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+case $CIRCLE_NODE_INDEX in
+  0 )
+    bundle exec rake test
```

Comment:
> if this is the "good way" to do it on Circle, can we not use Travis now?

Travis build matrix seems better than Circle CI matrix so far, and it has the advantage that every single OSS project on GitHub use it. If you guys want to give travis a try I can work on it so we can see how it looks like.